ArduPlane/Plane.h

```diff
@@ -1129,7 +1129,13 @@ class Plane : public AP_Vehicle {
 
     CrowMode crow_mode = CrowMode::NORMAL;
 
-    bool flare_switch_active;
+    enum class FlareMode {
+        FLARE_DISABLED,
+        ENABLED_NO_PITCH_TARGET,
+        ENABLED_PITCH_TARGET
+    };
+
+    FlareMode flare_mode = FlareMode::FLARE_DISABLED;
```

Comment:
just set FLARE_DISABLED=0 in the enum declaration, then no need to initialise flare_mode (saves a few bytes of flash)

Reply:
done